### ISSUE TITLE
Starfinder: Theme skill and skill synergy conflict

### ISSUE THREAD

**Issue by areichgott** [PCGen/pcgen#3447]
When my Envoy takes Xenoseeker as a theme in order to get Life Science as a class skill, PCGen does so properly. However, when my Envoy also takes the Skill Synergy feat with Life Science as one of the bonuses, the skill is not getting a +2 bonus on top of the class skill from earlier.

**Comment by LegacyKing** [comment]
I'm getting a +3 total bonus; which is how the rules are written.
Theme +1 "untyped", Class Skill +2 "Insight", Skill Synergy Feat +2 "Insight". Named bonuses do not stack per the rules. Reddit user Vehement1 posted this <https://www.reddit.com/r/starfinder_rpg/comments/6vzvbc/do_insight_bonuses_to_skills_stack/> - Page 266 under the heading "Bonuses" "When multiple bonuses apply to the same value different types of bonuses all apply, but in most cases bonuses of the same type do not add together (or "stack" with each other), unless a source specifies otherwise."

And here is  Owen K. C. Stephens from Paizo explaining it <http://paizo.com/threads/rzs2ui01&page=2?Starfinder-Early-Impressions#88> "At the same level the operative's edge hits +3 (the same bonus as Skill Focus), they get an ability (specialization skill mastery) that allows them to take 10 with any skill they have Skill Focus with, even if its not normally allowed. 
It's not a dead feat. Its benefit just becomes something unique only operatives have."

**Comment by areichgott** [comment]
Sorry, I may have been unclear in my original post. None of the bonuses that I'm talking about are the same type of bonus. I took the Xenoseeker theme not to get the +1, but to get the class skill, as Envoys do not get Life Science as a class skill. I then took Skill Synergy, which should give it a +2 insight bonus on top of that. Class skill bonuses are untyped.

> Class skills in which you have at least 1 rank are known as trained class skills; you gain a
+3 bonus to skill checks with such skills. (Core Rulebook p132)
